### ISSUE TITLE
Problem with Classifier Chain Implementation

### ISSUE THREAD

**Issue by pawanjay176** [u1234x1234/kaggle-yelp-restaurant-photo-classification#1]
Hi,

I am trying to use your classifier chain implementation on my set of features and using an xgboost classifier instance. I have double checked my training feature set and the training labels that I pass to the classifier chain instance. But, the predictions by using classifier chains drastically reduces. If I just use a simple xgboost classifier without chaining them, I get an F1 score of around 0.80 both on training and testing set. But, by using classifier chains, my F1 score goes down to 0.50 which is really low. Any thoughts on why would this be?

Thanks,
Pawan

**Comment by u1234x1234** [comment]
Hi Pawan,

I've used slightly modified version Classifier Chain from [scikit-learn PR](https://github.com/scikit-learn/scikit-learn/pull/3727). Some kind of problems may be due to the type of predictions: 0,1 labels, or probability. Could you post your code?
